.github/workflows/cron-job.yml

```diff
@@ -0,0 +1,32 @@
+name: Cron
+
+on:
+  schedule:
+    - cron: 0 * * * *
+  workflow_dispatch:
+  pull_request:
+
+env:
+  DISCORD_BOT_TOKEN: ${{ secrets.DISCORD_BOT_TOKEN }}
+  DISCORD_CHANNEL_ID: ${{ vars.DISCORD_CHANNEL_ID }}
+  DISCORD_MESSAGE_ID: ${{ vars.DISCORD_MESSAGE_ID }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_ORG: ${{ vars.GH_ORGANIZATION }}
+  TZ: Asia/Jakarta
+
+jobs:
+  run-go-script:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.22"
+
+      - name: Build
+        run: go build -o bin/main cmd/main.go
+
+      - name: Run
+        run: ./bin/main
```

Comment:
Are we going to use GitHub action to run this?

Reply:
yes, so far its the easiest way to run this, any problem in this?

Reply:
There's nothing wrong with it, but whenever there's a pull request, the update will be sent up to 1 hour later. And if we're going to increase the frequency, it may potentially spam the workflow run.

Reply:
got it, the `pull_request:` part was meant for the testing here since this is one way to trigger the action, will remove from this PR

Reply:
no no no, i mean the cron pattern `0 * * * *`, it's every hour right? it means it will only check every hour, so the update are not going to send immediately, only every 1 hour. But if we're going to increase the frequency, it may potentially spam the workflow run.

Reply:
oh, I think that one is reasonable frequency, let's see first how's this going and improve later if needed